### DISCUSSION TITLE
Command Pallete keyboard shortcut feedback

### DISCUSSION THREAD

**Discussion by mystiquewolf** [community/community#15283] [Other Feature Feedback, Questions, & Ideas] [answered]
Not sure what this feature is about, just coming to say that both `Ctrl` + `K` and `Ctrl` + `Alt` + `K` are taken by default, first one from Firefox search bar functionality and the second one from KDE Plasma "change keyboard input language".

TL;DR: Probably allow customization of the shortcuts...

**Comment by queenofcorgis** [comment, marked as answer]
Hi @mystiquewolf! Thanks for the feedback, we have a feature that allows you to customize the keyboard shortcuts used to open the command palette. You can edit your shortcuts in the [Accessibility section](https://github.com/settings/accessibility) of your user settings.
